default/python/AI/ColonisationAI.py

```diff
@@ -57,7 +57,18 @@
 
 
 def colony_pod_cost():
-    return AIDependencies.COLONY_POD_COST * (1 + state.get_number_of_colonies()*AIDependencies.COLONY_POD_UPKEEP)
+    if tech_is_complete(AIDependencies.GRO_LIFE_CYCLE):
+        base_cost = AIDependencies.CRYONIC_COLONY_POD_COST
```

Comment:
Have you added CRYONIC_COLONY_POD_COST constant?   

Nevermind, lets extract this value from ship part.